src/Parameters/DocumentParametersTrait.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+/*
+ * BigBlueButton open source conferencing system - https://www.bigbluebutton.org/.
+ *
+ * Copyright (c) 2016-2024 BigBlueButton Inc. and by respective authors (see below).
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation; either version 3.0 of the License, or (at your option) any later
+ * version.
+ *
+ * BigBlueButton is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with BigBlueButton; if not, see <https://www.gnu.org/licenses/>.
+ */
+
+namespace BigBlueButton\Parameters;
+
+use BigBlueButton\Core\Document;
+use BigBlueButton\Core\DocumentFile;
+use BigBlueButton\Core\DocumentUrl;
+use BigBlueButton\Enum\DocumentOption;
+use BigBlueButton\Parameters\Config\DocumentOptions;
+
+trait DocumentParametersTrait
```

Comment:
We had no issue with DocumentableTrait. Why did we change the name? Also if we can remove the "Trait" in the name. We should also keep the form of "Somethingable" recommended naming practice for traits.

Reply:
I just liked the other name more.... changed it back to the old name.